src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -120,6 +120,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@SteveL-MSFT, your last commit had 14 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

ConsoleHost unit tests.CustomPipeName startup tests.Should create pipe file if CustomPipeName is specified
```powershell
Expected $true, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 757
757:             & $powershell -CustomPipeName $pipeName -Command "Test-Path '$pipePath'" | Should -BeTrue
```
PSPath to native commands.PSPath with ~/path works
```powershell
Expected 0, but got 1.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\NativeExecution\NativeCommandArguments.Tests.ps1: line 107
107:         $LASTEXITCODE | Should -Be 0
```
PSPath to native commands.PSPath with ~ works
```powershell
Expected 0, but got 1.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\NativeExecution\NativeCommandArguments.Tests.ps1: line 113
113:         $LASTEXITCODE | Should -Be 0
```
PSModulePath changes interacting with other PowerShell processes.System32 module path prepended to PSModulePath.Allows Windows PowerShell subprocesses to load WinPS version of $PSHOME modules
```powershell
Expected like wildcard '*system32*' to match $null, but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 741
741:             powershell.exe -Command "Get-ChildItem | Out-Null;(Get-Module Microsoft.PowerShell.Management).Path" | Should -BeLike "*system32*"
```
Test suite for validating automounted PowerShell drives.Validating automounting FileSystem drives.Test automounting using subst.exe
```powershell
Expected exactly 'Drive found', but got @('Drive found', 'Invalid parameter - Y:\').
at <ScriptBlock>, D:\a\1\s\test\powershell\Provider\Pester.AutomountedDrives.Tests.ps1: line 40
40:            & $powershell -noprofile -command "& '$AutomountSubstDriveScriptPath' -FullPath '$substDir'" | Should -BeExactly "Drive found"
```